modules/artifact/src/test/java/org/mule/runtime/module/artifact/api/classloader/RegionClassLoaderTestCase.java

```diff
@@ -72,9 +72,17 @@
   private static final String ARTIFACT_ID = "testAppId";
   private static final String GROUP_ID = "com.organization";
   private static final String SPECIFIC_ARTIFACT_ID = "test-artifact";
+  private static final String SPECIFIC_ARTIFACT_ID_WITH_SPACES = "test-artifact-with-spaces";
   private static final String ARTIFACT_VERSION = "1.0.0";
   private static final String SPECIFIC_RESOURCE_FORMAT = "resource::" + GROUP_ID + ":" + SPECIFIC_ARTIFACT_ID + ":%s:%s:%s:%s";
+  private static final String SPECIFIC_RESOURCE_FORMAT_WITH_SPACES =
+      "resource::" + GROUP_ID + ":" + SPECIFIC_ARTIFACT_ID_WITH_SPACES + ":%s:%s:%s:%s";
   private static final String API_RESOURCE_NAME = "test-api.raml";
+  private static final String API_RESOURE_NAME_WITH_SPACES = "raml with spaces.raml";
+  private static final String API_FOLDER_NAME_WITH_SPACES = "folder with spaces";
+  private static final String API_RESOURCE_NAME_WITH_SPACES_ENCODED = API_RESOURE_NAME_WITH_SPACES.replace(" ", "%20");
```

Comment:
Where are these used?

Reply:
I added two tests, one that looks for a resource with spaces in the root of the artifact and another one that looks inside a folder which also has spaces just in case.
The ones with spaces are used to retrieve the resource using the resource:: syntax in which the spaces should be present. The ones with the %20 are used to build the URL which is expected to be returned by the classloader.
See: https://github.com/mulesoft/mule/pull/7647/files#diff-c002f1fe63a6bc2918bba26dc41b9f16R512
And: 
https://github.com/mulesoft/mule/pull/7647/files#diff-c002f1fe63a6bc2918bba26dc41b9f16R505